src/pages/UploadModal.tsx

```diff
@@ -137,7 +139,7 @@ const UploadModal: React.FC = () => {
         ))}
       </List>
       <Box style={{ justifyContent: "center" }}>
-        <Button onClick={handleUpload}>Upload</Button>
+        {fileLoaded ? <Button onClick={handleUpload}>Upload</Button> : null}
```

Comment:
instead of `fileLoaded ? <Button onClick={handleUpload}>Upload</Button> : null`

you can just do `fileLoaded && <Button onClick={handleUpload}>Upload</Button>`